debian/control

```diff
@@ -21,7 +21,7 @@ Build-Depends: debhelper (>= 10), devscripts
 Standards-Version: 4.1.2
 
 Package: delphix-rust
-Architecture: amd64
+Architecture: any
```

Comment:
I'm pretty sure this won't work long term, but I'm good landing as-is.. we can fix it later.

E.g. I think once we're building both packages, I think we'll need the architecture to be accurate, in order for both packages to exist simultaneously in an APT repository (e.g. our upgrade image, or the repository we build and pass into appliance-build)..

otherwise, I don't think an "apt-get install" will be able to properly pull in the correct package, based on the architecture of the system the package is being installed on..

but, lets go ahead and move forward with this, since it'll help other parts of the project.. and we can figure out how to do this properly later.

Reply:
So AIUI the way `any` works is that the build process will output the package for a single architecture based on the build system and configuration, and it can make lots of different ones based on the system where the build happens. They get a suffix applied with the specific architecture. An architecture-independent package (where all platforms produce the same product with the same name) uses `all` instead. This is based on my reading of https://www.debian.org/doc/debian-policy/ch-controlfields#architecture , which could be wrong.

Reply:
For some evidence, with `any` in use I got as output `delphix-rust_1.0.0-delphix-2022.09.15.23_arm64.deb`, which is namespaced by architecture

Reply:
Great, thanks for looking into this! so, this is specifying the architecture of the source package.. and not the binary package.. so I think that makes sense..

the source package, being able to be compiled on "any" arch.. will the produce an arch-specific binary package..

thanks for the clarification.. this looks good to me.